pages/moduleDashboard.js

```diff
@@ -1,7 +1,22 @@
+import React, { useContext, useState, useEffect } from 'react'
+import { useAppContext } from '../context/app-context'
+
 import styles from '../styles/Home.module.scss'
 
 export default function moduleDashboard() {
+  const { sharedState, setSharedState } = useAppContext()
+  const [module, setModule] = useState({})
+
+  useEffect(() => {
+    console.log(sharedState);
```

Comment:
Let's just be aware of this console.log() in the useEffect() here

Reply:
Removed! I was using it for testing so we don't need it anymore.